src/genophenocorr/preprocessing/_config.py

```diff
@@ -241,13 +241,15 @@ def load_phenopacket_folder(
     # Turn phenopackets into a cohort using the cohort creator.
     # Keep track of the progress by wrapping the list of phenopackets
     # with TQDM 😎
-    cohort_iter = tqdm(pps, desc='Patients Created')
+    cohort_iter = tqdm(pps, desc='Patients Created', file=sys.stdout, colour='green')
     notepad = cohort_creator.prepare_notepad(f'{len(pps)} phenopacket(s) found at `{pp_directory}`')
     cohort = cohort_creator.process(cohort_iter, notepad)
-
-
+    
     validation_summary = _summarize_validation(validation_policy, notepad)
```

Comment:
This is OK in principle, but I think it would be better to report *both* errors and warnings to STDERR (red), and report `No errors or warnings were found` to STDOUT (no red).
To do this, we would either need to return more data from `_summarize_validation`, or to deal with reporting the results within `_summarize_validation`. I think we should do the latter.

To make this happen, let's return nothing from `_summarize_validation` and deal with printing to streams within the function body. More info follows below.